src/elements/header/header-environment/readme.md

```diff
@@ -0,0 +1,38 @@
+The `sbb-header-environment` component displays a ribbon inside the header to indicate the current environment.
+
+**Note**: For the production environment, the sbb-header-environment is expected to be hidden.
+
+```html
+<sbb-header>
+  ...
+  <sbb-header-environment>dev</sbb-header-environment>
+</sbb-header>
+```
+
+## Style
+
+We provide default colors for `dev`, `edu`, `int`, `loc` and `test`. Any other environment is by default of color granite.
+
+It is possible to override the ribbon background color by overriding the `--sbb-header-environment-color` css var.
+Furthermore, the `--sbb-header-environment-text-color` var can be used change the text color.
+
+```scss
+sbb-header-environment {
+  --sbb-header-environment-color: custom-color;
+}
+```
+
+<!-- Auto Generated Below -->
+
+## CSS Properties
+
+| Name                                  | Default                  | Description                          |
+| ------------------------------------- | ------------------------ | ------------------------------------ |
+| `--sbb-header-environment-color`      | `var(sbb-color-granite)` | Can be used change the ribbon color. |
```

Comment:
I have one last thought, whether it's better to have "--sbb-header-environment-background-color" and "--sbb-header-color" to be nearer the property name. What do you think?